soc/riscv/riscv-privilege/nuclei/gd32vf103/Kconfig.defconfig.series

```diff
@@ -45,5 +45,8 @@ config PINCTRL_GD32_AFIO
 config GD32_HXTAL_VALUE
 	default 8000000
 
+config USART_GD32
+	default y
+	depends on SERIAL
```

Comment:
Drop this, this is selected by devicetree at Kconfig.gd32

Reply:
I'll apply it.